lib/resty/lmdb.lua

```diff
@@ -2,47 +2,111 @@ local _M = {}
 
 
 local transaction = require("resty.lmdb.transaction")
+local prefix = require("resty.lmdb.prefix")
 local status = require("resty.lmdb.status")
 
-do
-    local CACHED_TXN = transaction.begin(1)
 
+local next = next
```

Comment:
According to luacheck, `next` and `error` are never used in the code.

Reply:
Removed. Good catch!